readme.md

```diff
@@ -14,6 +14,17 @@
 
 ## Family
 
+### [Ghost Blitz](<https://boardgamegeek.com/boardgame/83195/ghost-blitz>)
+
+> Balduin, the house ghost, found an old camera in the castle cellar. Immediately he photographed everything that he loves to make disappear when he is haunting  – including himself, of course. Unfortunately, the enchanted camera takes many photos in the wrong colors. Sometimes the green bottle is white, at other times it's blue. Looking at the photos, Balduin doesn't really remember any more what he wanted to make disappear next. Can you help him with his haunting and quickly name the right item, or even make it disappear by yourself? If you grab the right items quickly, you have a good chance of winning...
+
+![Ghost Blitz game image](https://cf.geekdo-images.com/itemrep/img/D-2aizsXhm-kpy0ffA4R--Xd07Y=/fit-in/246x300/pic2034769.jpg)
+
+| Players | Min. Age |   Time |
+| ------: | -------: | -----: |
+|   2 - 8 |        8 |    20m |
```

Comment:
It looks like the linter is complaining about the formatting of the markdown cell. 🤦‍♂️

It should probably be condensed in that last column, to look more like:

```md
| Players | Min. Age | Time |
| ------: | -------: | ---: |
|   2 - 8 |        8 |  20m |
```

Reply:
Change done.